Javascript/100-Problems/script.js

```diff
@@ -850,3 +850,472 @@ const studentWithLowestPercentageInSubject = (subjectName) => {
 };
 
 console.log(studentWithLowestPercentageInSubject("Maths"));
+
+//function to find and print the subject(s) with the highest/lowest percentage of marks for a specific student
+
+const printSubjectPercentageOfStudent = (
+  studentName,
+  option,
+  obtainedPercentage
+) => {
+  const students = classObj.students;
+  const totalmark = 50;
+  let subjectName = "";
+  let subjectArray = [];
+  let subjectMark = 0;
+
+  students.find((student) => {
```

Comment:
don't use find for big conditions (find function will return the founded result but here we are not using it right?.)